tuner/tuner/libtuner.py

```diff
@@ -347,6 +360,10 @@ class ExecutionPhases(str, Enum):
     benchmark_models = "benchmark-models"
 
 
+class OutputFormats(str, Enum):
+    json = "json"
```

Comment:
Should I follow the Google Coding Standards and make this enum's values CapWords as well as the other enums in `libtuner.py`? I didn't use the CapWord style because it seemed out of place compared to the other enums in the libtuner

Also, should we keep the enum? I understood that we will only treat json, but this could help with extending support in the future, should I just remove it because it's not needed?

Reply:
I don't think we need it